jdi-light-material-ui-tests/src/main/java/io/github/com/pages/inputs/FabPage.java

```diff
@@ -0,0 +1,53 @@
+package io.github.com.pages.inputs;
+
+import com.epam.jdi.light.elements.common.UIElement;
+import com.epam.jdi.light.elements.composite.WebPage;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.UI;
+import com.epam.jdi.light.ui.html.elements.common.*;
+
+import java.util.List;
+
+public class FabPage extends WebPage {
+
+    @UI("//h2[text()='Basic buttons']/following-sibling::div[@id='basicActionBtns']/button")
+    public static List<Button> basicBtns;
+
+    @UI("//h2[text()='Basic buttons']/following-sibling::div/button[contains(@class,'MuiFab-primary')]")

```

Comment:
please make locator more compact, for example:
//button[@aria-label='add']
same for others